godel_process_planning/include/godel_process_planning/godel_process_planning.h

```diff
@@ -0,0 +1,35 @@
+#ifndef GODEL_PROCESS_PLANNING_H
+#define GODEL_PROCESS_PLANNING_H
+
+#include "godel_msgs/BlendProcessPlanning.h"
+#include "godel_msgs/KeyenceProcessPlanning.h"
+
+#include <descartes_core/robot_model.h>
+#include <pluginlib/class_loader.h>
+
+namespace godel_process_planning
+{
+
+class ProcessPlanningManager
+{
+public:
+  ProcessPlanningManager(const std::string& world_frame, const std::string& blend_group,
+                         const std::string& blend_tcp, const std::string& keyence_group,
+                         const std::string& keyence_tcp, const std::string& ik_plugin);
```

Comment:
`ik_plugin` refers to the plugin associated with `descartes_core::RobotModel`, not necessarily MoveIt's own ik plugin as specified inside of the `kinematics.yaml` config file. It may be worth changing this name and it's related uses to make this distinction clear.